app/stories/mockNextRouter.ts

```diff
@@ -0,0 +1,24 @@
+import {UrlObject} from 'url';
+import Router, {Router as RouterType} from 'next/router';
+import {action} from '@storybook/addon-actions';
+
+declare type Url = string | UrlObject;
+
+/* eslint-disable @typescript-eslint/consistent-type-assertions */
```

Comment:
:scream_cat:

Reply:
I know, monkeypatching makes typescript sad. I'm not sure how else to mock this though

Reply:
omg I'm an idiot. fixed